flybywiresim/a32nx
# [LIGHTS] Interior Lights - Big Changes & Fixes

# Summary of Changes

## Interior Lighting Improvements

- [x] FCU Display Backlighting
- [x] ATC Displays Backlighting
- [ ] Squawk Code Text Improvements

+ More Incoming

## Interior Lights (Fixes)

- [x] Chronometer
- [x] Rudder Trim
- [x] "PULL STD" Text
- [ ] TCAS Backlight

+ More Incoming

# Screenshots
## PULL STD
![Screenshot_235](https://user-images.githubusercontent.com/43104362/103713335-c3143d80-4fc4-11eb-8a08-f92d13c4b2dc.png)

## RUDDER TRIM
![Screenshot_236](https://user-images.githubusercontent.com/43104362/103713353-c90a1e80-4fc4-11eb-87c2-a65576aeebb1.png)

## FCU Backlighting
![Screenshot_234](https://user-images.githubusercontent.com/43104362/103713387-dde6b200-4fc4-11eb-90cf-9f0c04418ad2.png)

# How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

    1. Make sure you are signed in to GitHub
    2. Click on the Checks tab on the PR
    3. On the right side, click on the Artifacts drop down and click the A32NX link
